modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentEditor.es.js

```diff
@@ -8,17 +8,15 @@ import './FragmentPreview.es';
 import './SourceEditor.es';
 
 /**
- * Component that allows editing an existing or new Fragment
- * It integrates three <SourceEditor /> components for each part of
- * the fragment and a <FragmentPreview /> component for the preview
- * @review
+ * Creates a Fragment Editor Component that lets you edit an existing fragment 
+ * or a new Fragment. It integrates three `<SourceEditor />` components for each 
```

Comment:
@mwilliams2014 can the first sentence be changed to this:
> ... lets you create a new fragment or edit an existing fragment.

The way it's currently written says we can edit an existing or new fragment. This doesn't make sense to me because both are technically *existing*. I imagine it means *creating* a new fragment, but I wanted to check.

Reply:
I think you're right. It has to be creating a new fragment(which includes editing it) or editing an existing one. Sounds good.